concepts/eventing/performance evaluation/nats-jetstream/different-cloud-providers/results.md

```diff
@@ -0,0 +1,128 @@
+# Performance Evalation of Eventing on different Cloud Providers
+
+## Table of Contents
+- Test Setup
+- Failure Types
+- Test Results
+- Screenshots of Common Cases
+
+## Test Setup
+* Kyma CLI version: `2.6.2`
+* Kyma: 
+  * Production Profile
+  * JetStream with File Storage
+* K8s cluster:
+  * Kubernetes v1.24.3, v1.24.6
+  * Gardener cluster [Nodes: 2(min) to 6(max)]
+
+* Kyma deploy command:
+    ```
+    kyma deploy --source=${kyma_version} --profile production --value eventing.nats.nats.jetstream.fileStorage.storageClassName=${storage_class_name}
+    ```
+* Testing tool: [K6](https://k6.io/) (deployed in the same Kyma cluster)
+* Test duration is 2 days
+
+## Failure Types:
+Here are the common crash failures occured during the testing:
+- F1  - Only one of the NATS Jetstream nodes storage is full
+- F2  - All JetStream node storage is full
+- F3  - NATS JetStream just doesn’t accept and dispatch events despite storage is not full. This is solely observed on Azure
+- F4  - K6 stopped sending events in less than 20 hours
```

Comment:
If you have any error logs from EPP/EC/NATS, please put them in a follow-up ticket so we can investigate.